src/Device/jprogOperations.ts

```diff
@@ -94,7 +94,12 @@ export const verifySerialPortAvailable = (device: Device) => {
             reject();
             return;
         }
-        const serialPort = new SerialPort(device.serialport?.comName, {
+        const serialPort = new SerialPort({
+            path: device.serialport?.comName,
+            // NOTE: v8 of serialport had default baudRate 9600, but most of our devices
+            // defaults to 115200. Is there any please where the baudRate was set before,
+            // or did it only use the default?
+            baudRate: 115200,
```

Comment:
Unless we know for sure the baud rate of the device a larger or smaller baud rate will still mean we have 'corrupted' communication data. Hance the default value should be the most common one

Reply:
In this case it does not seem to matter what the baudRate is, since it only opens the port to check if JLink is working. I've set the baudRate to the old default, just to keep it consistent.